EvidenceApi.Tests/TestDataHelper.cs

```diff
@@ -73,6 +73,16 @@ public static DocumentType StaffSelectedDocumentType(string id)
             return reader.GetData().SelectMany(t => t.DocumentTypes).ToList().Find(dt => dt.Id == id);
         }
 
+        public static DocumentType GetStaffSelectedDocumentTypeByTeamName(string id, string teamName)
```

Comment:
I added this helper in because the test in `EvidenceApi.Tests/V1/E2ETests/DocumentSubmissionsTest.cs` was only pulling the first document type it found, instead of the document type of the right team. (E.g. two teams can have 'drivers-licence' as an ID, but the title might be different)